DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -98,8 +101,20 @@ function PreFirstEntryMapFixes()
             foreach AllActors(class'#var(prefix)BlackHelicopter',jock,'Helicopter'){break;}
             hoverHint = class'DXRTeleporterHoverHint'.static.Create(self, "", jock.Location, jock.CollisionRadius+5, jock.CollisionHeight+5, exit.Name);
             hoverHint.SetBaseActor(jock);
+        }
 
+        foreach AllActors(class'#var(prefix)SecurityBot2',sb,'enemy_bot') {
+            hoverHint = class'DXRHoverHint'.static.Create(self, "MJ12 Security Bot " $ ++botNum, sb.Location, sb.CollisionRadius*1.1, sb.CollisionHeight*1.1, sb.Name);
+            hoverHint.SetBaseActor(sb);
+            hoverHint.VisibleDistance = 15000;
```

Comment:
are you sure 15,000 is still needed with the other fixes that have happened? try commenting out this line to test it
```suggestion
            //hoverHint.VisibleDistance = 15000;
```

Reply:
actually the default is only 2000, probably 8000 would be good enough, but I guess 15000 is probably fine

Reply:
I indeed tested it. Using generous rounding, I estimated that the distance the furthest corners of the map is ~13000. I *really* want to see these labels.